hack/patch-converter-runner/README.md

```diff
@@ -0,0 +1,46 @@
+# Patch Converter Runner 
+
+## Overview
+
+The `patch-converter-runner` program invokes converter library to generate shoot from runtime resources. The purpose of this application is to allow developers to:
+- quickly reproduce problems with updating existing Gardener clusters
+- debug converter without a need to run the whole KIM system
+
+## Building the Binary
+
+To build the `patch-converter-runner` binary navigate to the `hack` folder, and execute the following command:
+
+```sh
+  go build -o ./bin/patch-converter-runner ./patch-converter-runner/cmd/main.go
```

Comment:
This results to check for consistency of the main vendor directory of the project. 

`go: inconsistent vendoring in /Users/I523107/GoPath/src/github.com/kyma-project/infrastructure-manager:
`
 I suggest running build command inside patch-converter-runner directory: 
```suggestion
binary navigate to the `hack/patch-converter-runner` folder, and execute the following command:

  go build -o ./bin/patch-converter-runner ./cmd/main.go
```